analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/SymbolReferenceAnalyzerTest.cs

```diff
@@ -101,7 +101,7 @@ public class SymbolReferenceAnalyzerTest
         [DataRow(ProjectType.Product)]
         [DataRow(ProjectType.Test)]
         public void Verify_NamedType_CS(ProjectType projectType) =>
-            Verify("NamedType.cs", projectType, 4, 3, 7, 7); // 'var' and type name on the same line
```

Comment:
"var" here was redirecting to the underlying type, which was nice. I've dropped the support, for now, to keep the logic simple but I might change my mind until if finish with this rework.